deeprank2/neuralnets/gnn/foutnet.py

```diff
@@ -13,8 +13,7 @@
 class FoutLayer(nn.Module):
     """FoutLayer.
```

Comment:
The first line of the docstring has no additional information to the name of the class.

Reply:
Edited to "Architecture based on multiple :class:`GNNLayer` layers."